atlp-rwanda/cabal-phantom-backend
# #ZM0sHy6Z Fixing sequelize setup

### **What about this PR?**
This PR is about to fix setup sequelize by making migrations and seeders untracked

### **Description of Task to be completed**
- Make migrations and seeders untracked

### **How should be manually tested?**
- Clone the repo
- Check if migrations and seeders are included in `.gitignore` that could make them untracked.

### **Relevant pivots tracker**
[#ZM0sHy6Z](https://trello.com/c/ZM0sHy6Z/17-configure-the-app-to-use-postgres-database)
